src/deckardcain.js

```diff
@@ -20,7 +20,7 @@ const REFRACT_PARSE_RESULT_ELEMENT_YAML = /[\uFEFF]?\n?\s*element: ?["']parseRes
  * @returns {string|null} Media type of given file.
  */
 function identify(source) {
-  if (source.match(API_BLUEPRINT_HEADER)) {
+  if (source.substring(0, 11).match(API_BLUEPRINT_HEADER)) {
```

Comment:
But regex we are using everywhere IS checking beginning of the string. And afaik we have no idea what would actually happen to our tools if you'd pass invalid, according to spec, APIB.

Only false negative scenario thing that _could_ happen is FORMAT not being first. But all our docs and examples have it at the beginning of document.